wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
Yes, it's not a heavy operation:
```Julia
julia> @time A = Array(UInt8, 10000)
  0.000009 seconds (157 allocations: 52.042 KB)
```
And the computer in use is not the best ever (a tablet suffering from the heat).

That's not that that will makes the wrapper becoming heavy.

And about the idea to keep a single buffer of this size? That makes also the error handling benefit from this larger buffer.